ONSdigital/rm-survey-service
# Database rationalisation

# What and why?

The survey service's database columns needed underscores.

# How to test?

Run acceptance tests and check that survey-related stuff on frontstage/response-ops work. Also, check report chasing link on response-ops just to make sure nothing's wrong with reporting service.

# Trello
[Card](https://trello.com/c/sX0mDu23)
